### DISCUSSION TITLE
Publish v3

### DISCUSSION THREAD

**Discussion by njfamirm** [11ty/eleventy#3104] [Uncategorized]
Hi, thanks for support esm by default!

Could you please publish `3.0.0-alpha.1` for real testing in my project.

the early bird gets the worm, the alpha bird gets the bugs 😂

**Comment by alimd** [comment]
Any news?

**Reply by alimd** [reply]
@zachleat

**Comment by wommy** [comment]
+1

i remember having to `npm install @11ty/eleventy@canary` for the longest while to use 2.0 features,
dunno why that wouldnt work here

**Reply by pdehaan** [reply]
That won't work since `canary` tags still point to v2.

```sh
npm info @11ty/eleventy dist-tags --json

{
  "latest": "2.0.1",
  "beta": "2.0.1-beta.2",
  "canary": "2.0.2-alpha.2"
}
```

**Reply by wommy** [reply]
im not saying they work, im saying it wouldnt be hard for them to work again

if anything im wondering why they stopped working

**Comment by pdehaan** [comment]
You should be able to install this directly from GitHub, 
```sh
npm i github:11ty/eleventy -S

added 151 packages, and audited 152 packages in 16s

31 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

```sh
node ./node_modules/.bin/eleventy --version
3.0.0-alpha.1
```

**Comment by njfamirm** [comment]
Thanks
